packages/bpk-mixins/src/mixins/_forms.scss

```diff
@@ -660,7 +660,7 @@
 
 @mixin bpk-radio__input {
   position: absolute;
-  top: ($bpk-spacing-base - $bpk-spacing-md) / 2;
+  top: $bpk-spacing-sm / 5;
```

Comment:
Same as above

Reply:
It is for the correct spacing that matches the Sketch libraries that Will has done as part of the type ramp changes